com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -168,7 +168,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed default value of "Distortion Blur" from 1 to 0 according to the doc.
 - Fixed Transparent Depth Pre/Post pass by default for the built-in HDRP Hair shader graph.
 - Fixed NullReferenceException when opening a Volume Component with a Diffusion Profile with any inspector.
-- 
+-
```

Comment:
Please undo this change. I'd like to keep this branch from changing anything outside of our folders so that it's easier to maintain until it's finally merged.